packages/ui-toolkit/package.json

```diff
@@ -81,17 +77,15 @@
     "redux": "^3.7.2",
     "redux-form": "^7.0.3",
     "snapguidist": "^2.0.0",
-    "stylelint": "^8.0.0",
-    "stylelint-config-primer": "^2.0.1",
-    "stylelint-config-standard": "^17.0.0",
-    "stylelint-processor-styled-components":
-      "styled-components/stylelint-processor-styled-components#2a33b5f",
+    "stylelint": "^8.1.1",
+    "stylelint-config-joyent-portal": "^1.0.1",
     "svgo": "^0.7.2",
     "tinycolor2": "^1.4.1",
     "title-case": "^2.1.1",
     "webpack": "^3.5.5"
   },
   "peerDependencies": {
+    "chart.js": "^2.7.0",
```

Comment:
Why was this added ?

Reply:
because it is needed by the metrics components

Reply:
Okay , carry on then :p